test_podman_compose_extends_w_file_subdir fails

**Describe the bug**
I forked the repository yesterday and found that the test cases on the `test_podman_compose_extends_w_file_subdir` test case is failing on it's final test assertion

```python
        # check container did not exists anymore
        out, err, returncode = capture(command_check_container)
        assert 0 == returncode
>       assert out == b""
E       assert b'"localhost/...ir_test:me"\n' == b''
E         Full diff:
E         - b''
E         + b'"localhost/subdir_test:me"\n'

tests/test_podman_compose.py:61: AssertionError
===================================================================== short test summary info =====================================================================
FAILED tests/test_podman_compose.py::test_podman_compose_extends_w_file_subdir - assert b'"localhost/...ir_test:me"\n' == b''
=================================================================== 1 failed, 1 passed in 4.74s ===========================================================
```
**To Reproduce**
Steps to reproduce the behavior:
1. what is the content of the current working directory (ex. `docker-compose.yml`, `.env`, `Dockerfile`, ...etc.)
N/A
2. what is the sequence of commands you typed
```shell
(venv) dwhitmire@MBP-YPW171W402 podman-compose % git remote -v
origin  git@github.com:dixonwhitmire/podman-compose.git (fetch)
origin  git@github.com:dixonwhitmire/podman-compose.git (push)
upstream        git@github.com:containers/podman-compose.git (fetch)
upstream        git@github.com:containers/podman-compose.git (push)
(venv) dwhitmire@MBP podman-compose % git branch
* devel
(venv) dwhitmire@MBP podman-compose % git merge upstream/devel
Already up to date.
(venv) dwhitmire@MBP podman-compose % python3 -m pytest
======================================================================= test session starts =======================================================================
platform darwin -- Python 3.10.5, pytest-7.2.0, pluggy-1.0.0
rootdir: /Users/dwhitmire/code/podman-compose
plugins: cov-4.0.0
collected 2 items                                                                                                                                                 

pytests/test_volumes.py .                                                                                                                                   [ 50%]
tests/test_podman_compose.py F                                                                                                                              [100%]
<remainder omitted for brevity>
```

**Expected behavior**
A clear and concise description of what you expected to happen.
The test case should pass.

**Actual behavior**
What is the behavior you actually got and that should not happen.


**Output**

```
$ podman-compose version
podman-compose version: 1.0.4
['podman', '--version', '']
using podman version: 4.3.0
podman-compose version 1.0.4
podman --version 
podman version 4.3.0

```

**Environment:**
 - OS: Linux / WSL / Mac
 - podman version: 
 - podman compose version: (git hex)

**Additional context**

It looks like there is mismatch between the image used in the [tests](https://github.com/containers/podman-compose/blob/devel/tests/test_podman_compose.py#L46) and the image within the [Dockerfile](https://github.com/containers/podman-compose/blob/devel/tests/extends_w_file_subdir/sub/docker/example/Dockerfile).

I will be interested to see if submitting a PR with the "quick fix" addresses the issue.

--- a/tests/test_podman_compose.py
+++ b/tests/test_podman_compose.py
@@ -43,7 +43,7 @@ def test_podman_compose_extends_w_file_subdir():
         "rmi",
         "--force",
         "localhost/subdir_test:me",
-        "docker.io/library/bash",
+        "docker.io/library/busybox",
     ]
 
     out, err, returncode = capture(command_up)

